app/components/subject-viewer.cjsx

```diff
@@ -7,19 +7,31 @@ getSubjectLocation = require '../lib/get-subject-location'
 FlagSubjectButton = require './flag-subject-button'
 SignInPrompt = require '../partials/sign-in-prompt'
 `import FrameViewer from './frame-viewer';`
-`import CollectionsManagerIcon from '../collections/manager-icon';`
```

Comment:
The staging link gives this error `Uncaught (in promise) ReferenceError: CollectionsManagerIcon is not defined`.